packages/slate-editor/src/icons/Component-Story-Bookmark.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="40" height="40" viewBox="0 0 40 40" xmlns="http://www.w3.org/2000/svg" fill="white">
+    <rect x="0.5" y="0.5" width="39" height="39" rx="5.5" stroke="#D1D5DB" />
+
+    <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M7.5 15.5V24.5H16.5V15.5H7.5ZM7 14C6.44772 14 6 14.4477 6 15V25C6 25.5523 6.44772 26 7 26H17C17.5523 26 18 25.5523 18 25V15C18 14.4477 17.5523 14 17 14H7Z"
+        fill="#4B5563" />
+    <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M34 18C34 18.2761 33.7761 18.5 33.5 18.5H22.5C22.2239 18.5 22 18.2761 22 18V17C22 16.7239 22.2239 16.5 22.5 16.5H33.5C33.7761 16.5 34 16.7239 34 17V18Z"
+        fill="#D1D5DB" />
+    <path fill-rule="evenodd" clip-rule="evenodd"
+        d="M30 23C30 23.2761 29.7761 23.5 29.5 23.5H22.5C22.2239 23.5 22 23.2761 22 23V22C22 21.7239 22.2239 21.5 22.5 21.5H29.5C29.7761 21.5 30 21.7239 30 22V23Z"
+        fill="#D1D5DB" />
+</svg>
```

Comment:
This is nitpicking. But still worth sharing, I believe.

Our current practice so far implied making SVG icons stylable with CSS. 

So instead of hard-coding colors, we use `currentColor`, plus allow to inherit `fill` style from above in case of bi-colored icons.

For example, see how Embed icon is implemented:

https://github.com/prezly/slate/blob/d31c4670af1fe6149c22066f04580766d85452fb/packages/slate-editor/src/icons/Component-Embed.svg

https://github.com/prezly/slate/blob/ab0c8ed76ec8d43e384ffb9ad59c7ac76b18c8d2/packages/slate-editor/src/modules/editor-v4-floating-add-menu/components/Dropdown.scss#L84-L95

```suggestion
<svg width="40" height="40" viewBox="0 0 40 40" xmlns="http://www.w3.org/2000/svg">
    <rect x="0.5" y="0.5" width="39" height="39" rx="5.5" stroke="currentColor" />

    <path fill-rule="evenodd" clip-rule="evenodd"
        d="M7.5 15.5V24.5H16.5V15.5H7.5ZM7 14C6.44772 14 6 14.4477 6 15V25C6 25.5523 6.44772 26 7 26H17C17.5523 26 18 25.5523 18 25V15C18 14.4477 17.5523 14 17 14H7Z"
        />
    <path fill-rule="evenodd" clip-rule="evenodd"
        d="M34 18C34 18.2761 33.7761 18.5 33.5 18.5H22.5C22.2239 18.5 22 18.2761 22 18V17C22 16.7239 22.2239 16.5 22.5 16.5H33.5C33.7761 16.5 34 16.7239 34 17V18Z"
        fill="currentColor" />
    <path fill-rule="evenodd" clip-rule="evenodd"
        d="M30 23C30 23.2761 29.7761 23.5 29.5 23.5H22.5C22.2239 23.5 22 23.2761 22 23V22C22 21.7239 22.2239 21.5 22.5 21.5H29.5C29.7761 21.5 30 21.7239 30 22V23Z"
        fill="currentColor" />
</svg>
```